modules/primer-popover/lib/popover.scss

```diff
@@ -0,0 +1,180 @@
+.Popover {
+  position: absolute;
+  z-index: 100;
+}
+
+.Popover-message {
+  position: relative;
+  width: 235px;
```

Comment:
I'd like to make this an even number, suggest making this `232px` or `24px`—something divisible by 8px because that's what our spacing scale is based on. I know you based this on the original custom code, let me know if you this would cause issues, if so, we can adjust in a follow-up relelase.